activation/handler_v2_test.go

```diff
@@ -932,6 +925,97 @@ func TestHandlerV2_ProcessMergedATX(t *testing.T) {
 		err = atxHandler.processATX(context.Background(), "", merged, time.Now())
 		require.ErrorIs(t, err, pubsub.ErrValidationReject)
 	})
+	t.Run("publishing two merged ATXs by one marriage set is malfeasance", func(t *testing.T) {
```

Comment:
NIT: I think it should be from instead of by

```suggestion
	t.Run("publishing two merged ATXs from one marriage set is malfeasance", func(t *testing.T) {
```